src/views/Products.jsx

```diff
@@ -1,7 +1,104 @@
-import React from 'react'
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import ProductsSidebar from '../components/ProductsSidebar'
+import ProductCategories from '../mocks/en-us/product-categories.json'
+import ProductsJson from '../mocks/en-us/products.json'
+import '../styles/Products.scss'
+import ProductItem from '../components/FeaturedProducts/ProductItem'
+import ProductsPagination from '../components/ProductsPagination'
+import Spinner from '../components/Spinner'
+
+export default function Products ({ onHandleLinkPage }) {
+  const [isLoading, setIsLoading] = useState(true)
+  const [filters, setFilters] = useState([])
+  const [filteredProducts, setFilteredProducts] = useState(ProductsJson.results)
+
+  const isInFilters = (value) => {
+    return filters.indexOf(value) !== -1
+  }
+
+  const filterProducts = () => {
+    const products = ProductsJson.results.filter((fProduct) => {
+      return filters.includes(fProduct.data.category.id)
+    })
+    setFilteredProducts(products)
+  }
+
+  const resetLoading = () => {
+    setTimeout(() => {
+      setIsLoading(false)
+    }, 2000)
+  }
```

Comment:
similar `isInFilters`, `resetLoading` can be wrapped in a `useCallback` to prevent unnecessary assignments